pkg/pillar/hypervisor/kvm.go

```diff
@@ -109,8 +109,6 @@ const qemuConfTemplate = `# This file is automatically generated by domainmgr
   driver = "intel-iommu"
   caching-mode = "on"
 {{ end }}
-[realtime]
```

Comment:
The replacement option for realtime is `overcommit mem-lock`, which is disabled by default, so I think we are fine to just remove it, but if you want to be on the safe side, just set off explicitly....
https://www.qemu.org/docs/master/about/removed-features.html#realtime-removed-in-6-0

Reply:
Explicitly turning off the overcommit mem-lock results in pillar error so I am completely removing the [realtime] parameter as of now.